src/main/kotlin/no/item/gradle/InitRunnerPlugin.kt

```diff
@@ -0,0 +1,19 @@
+package no.item.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class InitRunnerPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        val extension = target.extensions.create(
+            "initRunner",
```

Comment:
Lets rename `"initRunner"` to `"generateTypeScript"`.

Reply:
And also rename the files themselves.

Reply:
Good idea, implemented